### DISCUSSION TITLE
Extending VS Code Extension Manager

### DISCUSSION THREAD

**Discussion by alagiribuddhar** [microsoft/vscode-discussions#908] [Extension Development QnA] [answered]
I am having an use case where I am installing VS Code extension from local source. These extensions are located in my internal artifactory and not published to VS Code market place. Is there any way to extend VS Code extension manager to show updates from artifactory whenever new extension is available from my internal artifactory?

**Comment by gjsjohnmurray** [comment, marked as answer]
AFAIK this isn't currently possible.

You may like to comment on https://github.com/microsoft/vscode/issues/179919 and/or the (much older) https://github.com/microsoft/vscode/issues/21839

**Reply by alagiribuddhar** [reply]
Thank you. The mentioned GitHub issues were related to having private Extension Manager. My use case is about extending VS Code Extension Manager. Like having proxy or multiple VS Code endpoints.
